modules/claims_api/spec/lib/claims_api/v2/disability_compensation_pdf_mapper_spec.rb

```diff
@@ -54,6 +54,8 @@
 
       describe 'when the claimProcessType is BDD_PROGRAM' do
         let(:claim_process_type) { 'BDD_PROGRAM' }
+        let(:anticipated_seperation_date) { '2024-03-05' }
+        let(:active_duty_end_date) { '2024-03-05' }
 
```

Comment:
If I understand this correctly the claim_date is always `today` so these dates will need to be dynamically set to always be something like `today + 4.months` (sudo code).  Otherwise this will eventually start to fail whenever that date spot hits and it is no longer after the lower end date or between the two for the check later on.